diff --git a/contrib/download_prerequisites.old b/contrib/download_prerequisites
index 72976c4..1b77ed9 100755
--- a/contrib/download_prerequisites.old
+++ b/contrib/download_prerequisites
@@ -164,7 +164,7 @@ do
                     chksum='shasum -a 512 --check'
                   ;;
                   *)
-                    chksum='sha512sum --check'
+                    chksum='sha512sum -c'
                   ;;
                 esac
                 chksum_extension='sha512'
@@ -243,7 +243,7 @@ do
     package="${ar%.tar*}"
     if [ ${force} -gt 0 ]; then rm -rf "${directory}/${package}"; fi
     [ -e "${directory}/${package}" ]                                          \
-        || ( cd "${directory}" && tar -xf "${ar}" )                           \
+        || ( cd "${directory}" && tar -xf "${ar}" --no-same-owner )           \
         || die "Cannot extract package from ${ar}"
     unset package
 done
