diff --git a/src/extract.c.old b/src/extract.c
index 8276f8f..ed35e45 100644
--- a/src/extract.c.old
+++ b/src/extract.c
@@ -175,8 +175,6 @@ void
 extr_init (void)
 {
   we_are_root = geteuid () == ROOT_UID;
-  same_permissions_option += we_are_root;
-  same_owner_option += we_are_root;
 
   /* Option -p clears the kernel umask, so it does not affect proper
      restoration of file permissions.  New intermediate directories will
